### PR TITLE
Improvement/zenko 4616 mongodb 5.0

### PR DESCRIPTION
**Pull request by ghost**
To be merged when we have dedicated branch / pensieve API mongodb driver migration merged and released.

The PR is about upgrading the mongodb charts to support v5.0. There is also a need to update the init scripts for the shard server https://github.com/scality/Zenko/pull/1913/commits/3703a948ba7658950544ac2bf542f1791da44b4d

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VERSION="2.8.4"
+VERSION="2.9.0"
```
Comment (francoisferrand) [inline VERSION:1]: should not target 2.9 branch, do not forget to create it and retarget PR before merging.

(though we should not release zenko 2.9.0 right after : if you need to create a release afterwards for bumping in Artesca, should be a pre-release like `2.9.0-preview.1` or `2.9.0-alpha.1`)

Comment (ghost) [reply VERSION:1]: Yes, this is only for testing.

File: solution/deps.yaml [hunk 2]
```diff
@@ -47,9 +47,9 @@ kafka-lag-exporter:
   tag: 0.7.1
   envsubst: KAFKA_LAGEXPORTER_TAG
 pensieve-api:
-  sourceRegistry: registry.scality.com/pensieve-api
+  sourceRegistry: registry.scality.com/playground/wlardier
```
Comment (nicolas2bert) [inline solution/deps.yaml]: Reminder: don't forget to update it :)

**Review by nicolas2bert — APPROVED** [review]

File: solution/deps.yaml (hunk 2 above)
Comment (ghost) [reply solution/deps.yaml]: Yes! Thank you :slightly_smiling_face:

**Review by KillianG — APPROVED** [review]

File: solution/deps.yaml (hunk 2 above)
Comment (ghost) [reply solution/deps.yaml]: Updated and branch rebased to dev/2.9

File: VERSION (hunk 1 above)
Comment (ghost) [reply VERSION:1]: PR is now rebased on the latest dev/2.9 branch.

**Pull request merged**